osbuild/solver/dnf.py

```diff
@@ -230,7 +243,8 @@ def search(self, args):
         return packages
 
     def depsolve(self, arguments):
-        # # Return an empty list when 'transactions' key is missing or when it is None
+        want_sbom = "sbom" in arguments
```

Comment:
With the single use of `want_sbom` later on in this function we could skip this assignment and use `if "sbom" in arguments` directly in the conditional?

Reply:
Yes, this should be OK to do.